.github/workflows/abq-ci.yml

```diff
@@ -4,28 +4,31 @@ on:
 
 jobs:
   test_abq:
-    runs-on: macos-12
+    runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: false
       matrix:
         include:
           - name: abq
             version: "1.7.4"
+            os: macos-12 # intel
+          - name: abq
+            version: "1.7.4"
+            os: macos-14 # arm
+          - name: abq@1
+            version: "1.7.4"
+            os: macos-12 # intel
           - name: abq@1
             version: "1.7.4"
+            os: macos-14 # arm
     steps:
       - uses: actions/checkout@v3
-        with:
-          path: homebrew-fake-tap-name # give it a fake name so we don't accidentally pull the one from github
-
-      - name: move it to the right directory
-        run: |
-          mkdir -p homebrew-tap /usr/local/Homebrew/Library/Taps/rwx-research
-          mv homebrew-fake-tap-name /usr/local/Homebrew/Library/Taps/rwx-research/
 
       - name: Install abq & run it
         run: |
           set -o pipefail
-          brew install rwx-research/fake-tap-name/${{ matrix.name }} 2>&1 | tee install-output.txt
+          ! command -v abq
+          brew install ./Formula/${{ matrix.name }}.rb 2>&1 | tee install-output.txt
```

Comment:
I tweaked how this works a touch because these commands were failing on macos-14. `sudo` worked to create the dirs and move the tap, but then `brew install failed` due to permissions.

This seemed easier than resolving all those shenanigans.